src/containers/PlaylistLiveChannels/PlaylistLiveChannels.tsx

```diff
@@ -149,8 +156,7 @@ function PlaylistLiveChannels({ playlist: { feedid, title, playlist } }: { playl
             <>
               <StartWatchingButton
                 item={channelMediaItem}
-                playUrl={addQueryParams(`/p/${feedid}`, {
-                  play: 1,
+                playUrl={addQueryParams(liveChannelsURL(feedid || '', channelId, true), {
```

Comment:
Maybe extract this to a factory function

Reply:
I had that initially, but since it has multiple optional params: `start`, `end` and `beginning` the util became too long for my taste:

```tsx
                playUrl={liveChannelsURL(feedid || '', channelId, true, isVod ? program?.startTime : undefined, isVod ? program?.endTime : undefined)}
```

And for the "Start from beginning button":

```tsx
                onClick={() => history.push(liveChannelsURL(feedid || '', channelId, true, program?.startTime : undefined, undefined, true))}
```